rudder-rest/src/main/scala/com/normation/rudder/rest/RestExtractorService.scala

```diff
@@ -217,10 +217,10 @@ case class RestExtractorService (
   }
 
   private[this] def toWorkflowStatus (value : String) : Box[Seq[WorkflowNodeId]] = {
-    val possiblestates = workflowService.stepsValue
+    val possiblestates = workflowLevelService.getWorkflowService().stepsValue
```

Comment:
In this fonction, workflowLevelService.getWorkflowService() could be stored in a val at the beginning.

But I don't think it has much impact :)

Reply:
Hum. In fact, it may be the opposite. We need to ensure all over the place that workflow status (and so, the workflow we user anywhere) is dynamic and change it the user enable the license or the license become unavailable or if the user property "enable workflow" value is switched. So perhaps it should be a `def` here, I will look